_chapters/1.home.md

```diff
@@ -71,6 +71,24 @@ Test the asset against conventions held within an organisation to prevent bad co
 <br>
 <br>
 
+### What plugins are available?
+
+- Official plugins
+  - https://github.com/pyblish/pyblish-starter
+  - https://github.com/pyblish/pyblish-advance
```

Comment:
Advance and Flower were never really implemented.. I think we should discard those.

Reply:
Done :)